blueocean-i18n/src/main/java/io/jenkins/blueocean/i18n/BlueI18n.java

```diff
@@ -136,12 +136,19 @@ public HttpResponse doDynamic(StaplerRequest request) {
     private JSONObject getBundle(BundleParams bundleParams, Locale locale) {
         PluginWrapper plugin = bundleParams.getPlugin();
 
-        if (plugin == null) {
-            return null;
-        }
 
         try {
-            ResourceBundle resourceBundle = ResourceBundle.getBundle(bundleParams.bundleName, locale, plugin.classLoader);
+            ResourceBundle resourceBundle;
+            if (plugin == null) {
+                Jenkins jenkins = Jenkins.getInstance();
+                if (jenkins!=null && jenkins.pluginManager!=null && jenkins.pluginManager.uberClassLoader!=null) {
```

Comment:
I think this links in to my [comment](https://github.com/jenkinsci/blueocean-plugin/pull/967/files#diff-eb19343eb8fa1a34c9981c57f23d1f92L137) above.  Given that we could have clashes in jar resources no one says that within a jar resource everything needs to be unique) - looking in the uber class loader first could cause some issues...